charts/fluentbit/values.yaml

```diff
@@ -0,0 +1,158 @@
+namespace:
+  create : true
+  name: "calyptia-fluentbit"
+
+serviceAccount:
```

Comment:
I'm not sure we can create a service account, it needs to have the IAM role to register usage for marketplace products

Reply:
We should use a fixed name for the service account or require the user to provide it in advance.

Reply:
i think we may need to use the role arn of the service account created by the eksctl command in the kubernetes service account's annotation. let me check

Reply:
i have added the role ARN in .Values.serviceAccount.annotations and added the field for name in  .Values.serviceAccount.name.